packages/document/builder-doc/docs/en/config/tools/htmlPlugin.md

```diff
@@ -10,13 +10,17 @@ const defaultHtmlPluginOptions = {
   templateParameters, // corresponding to the html.templateParameters config
   chunks: [entryName],
   minify: {
-    removeComments: false,
+    removeComments: true,
     useShortDoctype: true,
     keepClosingSlash: true,
     collapseWhitespace: true,
     removeRedundantAttributes: true,
     removeScriptTypeAttributes: true,
     removeStyleLinkTypeAttributes: true,
+    removeEmptyAttributes: true,
+    minifyJS: true,
```

Comment:
Can we use the same terser config for both normal JS and html JS?

Reply:
I aligned minifyJS config, but minifyCSS is hard to change to cssnano